cypress.json

```diff
@@ -16,4 +16,4 @@
     "grepFilterSpecs": true,
     "grepOmitFiltered": true
   }
-}
+}
```

Comment:
Need this change?

Reply:
esLint magic. opened file itself, looks similar to the previuos version